### DISCUSSION TITLE
Coolapk Lite v0.0.3

### DISCUSSION THREAD

**Discussion by wherewhere** [Coolapk-UWP/Coolapk-Lite#65] [Announcements]
###### 更新日志
1. 更新版本号至 v0.0.3
2. 修复了一些 Bug
3. 优化了一些 UI
4. 现在可以显示更多卡片了
5. 现在可以使用更多多窗口了
###### 下载链接
[![Microsoft Store](https://img.shields.io/badge/download-下载-magenta.svg?label=Microsoft%20Store&logo=data:image/svg+xml;base64,PHN2ZyByb2xlPSJpbWciIHZpZXdCb3g9IjAgMCAyNCAyNCIgZmlsbD0id2hpdGUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHRpdGxlPk1pY3Jvc29mdCBTdG9yZTwvdGl0bGU+PHBhdGggZD0iTTExLjQgOS42djQuMkg3LjJWOS42aDQuMnptMCA5LjZWMTVINy4ydjQuMmg0LjJ6bTUuNC05LjZ2NC4yaC00LjJWOS42aDQuMnptMCA5LjZWMTVoLTQuMnY0LjJoNC4yek03LjIgNS40VjIuN2MwLTEuMTYuOTQtMi4xIDIuMS0yLjFoNS40YzEuMTYgMCAyLjEuOTQgMi4xIDIuMXYyLjdoNi4zYS45LjkgMCAwIDEgLjkuOXYxMy44YTMuMyAzLjMgMCAwIDEtMy4zIDMuM0gzLjNBMy4zIDMuMyAwIDAgMSAwIDIwLjFWNi4zYS45LjkgMCAwIDEgLjktLjloNi4zek05IDIuN3YyLjdoNlYyLjdhLjMuMyAwIDAgMC0uMy0uM0g5LjNhLjMuMyAwIDAgMC0uMy4zek0xLjggMjAuMWExLjUgMS41IDAgMCAwIDEuNSAxLjVoMTcuNGExLjUgMS41IDAgMCAwIDEuNS0xLjVWNy4ySDEuOHYxMi45eiIvPjwvc3ZnPg==&style=for-the-badge&color=11a2f8)](https://www.microsoft.com/store/apps/9NB8J1BH0D7T "Microsoft Store")
###### 屏幕截图
![主页](https://github.com/user-attachments/assets/885fb38c-380c-4af5-86f3-8f1b16349829)

<hr /><em>This discussion was created from the release <a href='https://github.com/Coolapk-UWP/Coolapk-Lite/releases/tag/v0.0.3'>Coolapk Lite v0.0.3</a>.</em>

**Comment by PermissionDog001** [comment]
这个版本加载图片失败
